sdk/containerinstance/arm-containerinstance/src/containerInstanceManagementClientContext.ts

```diff
@@ -13,7 +13,7 @@ import * as msRest from "@azure/ms-rest-js";
 import * as msRestAzure from "@azure/ms-rest-azure-js";
 
 const packageName = "@azure/arm-containerinstance";
-const packageVersion = "0.1.0";
+const packageVersion = "5.5.0";
```

Comment:
There's clearly breakingchange of this package. Please bump the major version. Thx